Assets/Scripts/PlayerCharacter/PlayerController.cs

```diff
@@ -6,17 +6,25 @@ namespace Cosmobot
     [RequireComponent(typeof(Rigidbody))]
     public class PlayerController : MonoBehaviour, DefaultInputActions.IPlayerMovementActions
     {
+        public enum RotationMode
+        {
+            MovementDirection,
+            CameraDirection
+        }
+
         public float moveSpeed;
         public float acceleration;
         public float jumpForce;
         public float gravity;
         public float maxFloorAngleDegrees;
         public float playerRotationSpeed;
+        public RotationMode rotationMode;
 
         public Transform cameraTransform;
         public Transform groundCheckOrigin;
```

Comment:
I think this should be private. We don't want any other script to change this value

```suggestion
        [SerializeField]
        private Transform cameraTransform;
        [SerializeField] 
        private Transform groundCheckOrigin;
```